app/posts/how-to-use-cross-government-slack.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-06-27
+tags: post
+layout: post
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
```

Comment:
Happy to add an illustration to this post; does anyone know how to add commits to a branch on a forked repo? Alternatively, can merge this in and add illustration later?